grain/_src/python/data_loader_test.py

```diff
@@ -44,6 +44,14 @@
 FLAGS = flags.FLAGS
 
 
+def setup_module():
+    # Set the path to test data when run via pytest. Otherswise,
+    # FLAGS.test_srcdir is set in `__main__` when run directly (or via bazel)
+    import grain
+    srcdir = pathlib.Path(grain.__file__).parents[0] / "_src" / "python"
+    FLAGS["test_srcdir"].parse(str(srcdir))
```

Comment:
This is kind of module-level fixtures meant to replace https://github.com/google/grain/pull/1013#pullrequestreview-3166495443